db/seeds.rb

```diff
@@ -22,7 +22,7 @@
     # ? Create 5 random posts for each user.
     5.times do
       new_post = Post.create(
-        author: user,
+        author_id: user.id,
         title: Faker::Lorem.sentence,
         text: Faker::Lorem.paragraph
       )
```

Comment:
- [ ] **[OPTIONAL]** Kindly fix your seed file errors. If you can't fix it then kindly remove all the data there.

  ![image](https://github.com/demesameneshoa/blog-app/assets/36542986/db960594-7460-4fef-9d20-dd07143c280b)

Reply:
Thank you!